### DISCUSSION TITLE
Working code and project timeline?

### DISCUSSION THREAD

**Discussion by JasperAlexander** [holtwick/briefing#123] [Q&A] [answered]
First of all, thank you for creating this repository!

Is there a working version or up-to-date documentation of this code? If not, when will this be finished?

**Comment by holtwick** [comment, marked as answer]
Thanks.

Yes, the current code on `master` should be working code.

You can find further documentation in the [docs](https://github.com/holtwick/briefing/tree/develop/docs) folder. More recent versions are always available on the `develop` branch. Here is a specific [instruction](https://github.com/holtwick/briefing/blob/master/docs/development.md) for developers.

It would be great if you could give me some examples where I could add or improve documentation. 

The README needs some improvements, I'll try to address this in the next days.
